app/scripts/background.js

```diff
@@ -28,6 +28,9 @@ const setupMetamaskMeshMetrics = require('./lib/setupMetamaskMeshMetrics')
 const EdgeEncryptor = require('./edge-encryptor')
 const getFirstPreferredLangCode = require('./lib/get-first-preferred-lang-code')
 const getObjStructure = require('./lib/getObjStructure')
+const regeneratorRuntime = require('regenerator-runtime');
```

Comment:
Why is this needed?

Reply:
Because of the lumino-light-client-sdk, it uses it when it gets compiled to transform async await operations, so basically if we don't define that in the background script we don't have it in the compiled version and when it's called it fails because it's undefined.